Source/Renderer/Context.js

```diff
@@ -191,7 +191,7 @@ define([
         this._canvas = canvas;
 
         options = clone(options, true);
-        options = defaultValue(options, {});
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
```

Comment:
This change is incorrect because the line below modifies `options`.

Reply:
So, I should change it back to `options = defaultValue(options, {});` ?

Reply:
I believe so, yes.  Please also put a comment pointing out that `options` is being modified, which explains why we can't use `EMPTY_OBJECT` here.

Reply:
Hi @shunter @emackey I have made the changes, is there anything else I need to change?